checador/app/assets/stylesheets/records.scss

```diff
@@ -11,4 +11,4 @@
     text-align: center;
     font-size: 25px;
   }
-}
\ No newline at end of file
+}
```

Comment:
Since you are creating this file [here](https://github.com/BC-MAY-21-ROR/kata-06-reloj-checador-k06-t05/pull/53/commits/dce39828bfe41220a5fcb5c2d121f90884ffa0b2), you don't need to create another commit to fix it, you should fix it in the same commit